templates/activity.mustache

```diff
@@ -74,11 +74,26 @@
         {{> filter_siyavula/mastery}}
     {{/sectionid}}
 
-    <div id="monassis" class="monassis monassis--practice monassis--maths monassis--siyavula-api">
+    {{#assignmentid}}
```

Comment:
I think these two wrappers could be combined, repeating the code makes it difficult to understand what's going on.